README.md

```diff
@@ -1,2 +1,36 @@
 # My_Portfolio
 The first contruction of my future professional portfolio.
+
+## Built With
+
+- HTML and CSS
+- Boilerplate for HTML
+- Linters for HTML and CSS
+
+## Getting Started
+
+>This repository is for setting up my first portfolio as a software developer. With this excercise I begun to build my main-page for showing my projects and my work on a friendly way.
+
+## Authors
+
+👤 **Marco Antonio Hincapié Montes**
+
+- GitHub: [@AntonioHincapie](https://github.com/AntonioHincapie)
+- Twitter: [@MarcoHincapie](https://twitter.com/MarcoHincapie)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/marco-hincapi%C3%A9-7a76751a3/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse curriculum.
+- This is Inspirated on Figma templeates.
+- My code partners and code reviewers. 
```

Comment:
- [ ] Please add the "License" section and the `MIT` file which is provided [here](https://github.com/microverseinc/readme-template) by Microverse.

Reply:
resolved 👍